packages/framework/src/Resources/views/Admin/Inline/Menu/menu.html.twig

```diff
@@ -1,6 +1,29 @@
 <ul class="js-toggle-menu navig navig--side">
 
     {% if isMultidomain() %}
+        <li class="js-toggle-menu-item navig__item">
+            <div class="box-dropdown">
+                <div
+                        class="box-dropdown__select box-dropdown__select--menu"
+                        title="{{ 'Switch locale'|trans }}"
+                >
+                    <i class="svg svg-flag"></i>
+                    <span class="box-dropdown__select__text">
+                        {{ localeFlag(app.user.selectedLocale) }}
+                    </span>
+                    <i class="box-dropdown__select__arrow box-dropdown__select__arrow--up"></i>
+                    <i class="box-dropdown__select__arrow box-dropdown__select__arrow--down"></i>
+                </div>
+                <ul class="js-toggle-menu-submenu box-dropdown__options">
+                    {% for locale in locales %}
+                        <li class="box-dropdown__options__item">
```

Comment:
I noticed that in the Czech localization, the text is still squeezed to the right. 
![Snímek obrazovky 2024-11-06 v 20 39 14](https://github.com/user-attachments/assets/4ba7122c-98f5-467c-832e-f57e97652c11)

You can add class `padding-right-10` here to improve the look